CONTRIBUTING.md

```diff
@@ -0,0 +1,31 @@
+# Contributing to Spark Design System
+
+## Type of change
+
+Contributions are divided into 2 types: **minor changes** and **substantial changes**.
+
+### Minor Changes
+
+Minor changes include, but are not limited to:
+
+* Bug fixes
+* Performance improvements
+* Accessibility improvements
+* Unit tests
+* Spelling and grammar fixes
+
+Minor changes should begin with an [issue on Github](https://github.com/sparkdesignsystem/spark-design-system/issue) if you are unable to make the change yourself or are unsure if the change you want to make is needed. If you make the changes yourself, you can [open a pull request](https://github.com/sparkdesignsystem/spark-design-system/pulls) for review.
+
+If a minor change turns out to be substantial, the Spark team may ask you to go through the contribution workflow for substantial changes instead.
+
+### Substantial Changes
+
+Substantial generally changes fall into 2 categories: New components and amendments to an existing component that change it's design or behavior. This type of change requires you to complete our contribution workflow to determine if a new component or change to an existing component aligns with the direction Spark is going.
```

Comment:
switch the words generally and changes here.

Reply:
it's should be its.

Reply:
it's should be its in this use case.

Reply:
For the curious: "it's" should only be used for a contraction, _it is_ or _it has_, where "its" is the possessive like _hers_ or _theirs_.